Extend panel guidance to advise not using long highlighted information (#3131)
alphagov/govuk-design-system: src/components/panel/index.md

{{ example({ group: "components", item: "panel", example: "default", html: true, nunjucks: true, open: false, size: "m", titleSuffix: "second" }) }}

### How to write panel text

> Keep your heading text brief, as it's only meant for a high-level explanation of what has happened. For example, 'Application complete'.

Reviewer: Should this also be changed from heading text to panel text?

> If you can, use short words and keep the highlighted information short. Shorter words are less likely to wrap around the panel, which can happen when users on mobile zoom in. Shorter highlighted information is also easier to read and remember.

Reviewer: ```suggestion
Aim to use short words and phrases to make sure highlighted information is easy to read in all formats. For example, shorter amounts of information is less likely to wrap around the panel, which can happen when using the zoom function on mobiles.
```

Reviewer: Trying a slightly different approach. Does it work and is it still accurate?

Author: I reckon so 👍🏻

> Aim to use short words and phrases to make sure highlighted information is easy to read in all formats. For example, shorter amounts of information is less likely to wrap around the panel, which can happen when using the zoom function on mobiles.

Reviewer 2: We tend to say "different screen sizes" elsewhere?

But "formats" for variations in postcodes, card details, telephone numbers etc

```suggestion
Aim to use short words and phrases to make sure highlighted information is easy to read at different screen sizes. For example, shorter amounts of information is less likely to wrap around the panel, which can happen when using the zoom function on mobiles.
```
